### PR TITLE
added python code for unused eip

### PR DESCRIPTION
**Pull request by cloudmlops**
- Adding changes to code for unused eip

### CODE REVIEW

File: lambda/unused_eip.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        if len(eip_list) != 0:
+            for eip in eip_list:
+                allocation_id = addresses['Addresses'][0]['AllocationId']
+                print("Allocation id is",allocation_id)
+                instance_id = addresses ['Addresses'][0]['InstanceId']
+                print("Instance Id is",instance_id)
+                network_interface_id = addresses['Addresses'][0]['NetworkInterfaceId']
+                print("NetworkInterfaceId",network_interface_id)
+        else:
+            print("No EIP Found")
```
Comment (swapnil-aws2022) [inline lambda/unused_eip.py:21]: change msg to NO eip found in this region

**Review by swapnil-aws2022 — CHANGES_REQUESTED** [review]
added reviwew comment on code

File: lambda/unused_eip.py (hunk 1 above)
Comment (cloudmlops) [reply lambda/unused_eip.py]: Changes are made as expected.

**Review by swapnil-aws2022 — APPROVED** [review]
changes looks fine

**Pull request merged**